### PR TITLE
ROX17656: Modified note in creating resources by using the init bundle section.

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.0+

Issue: [ROX-17656](https://issues.redhat.com/browse/ROX-17656)

Link to docs preview: [Docs Preview](https://63292--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/init-bundle-ocp#create-resource-init-bundle_init-bundle-ocp)

QE review:
- [ ] No QE check needed.

Additional information:

### CODE REVIEW

File: modules/create-resource-init-bundle.adoc [hunk 1]
```diff
@@ -50,7 +50,8 @@ endif::[]
 
 [NOTE]
 ====
-If you are installing by using Helm charts, do not perform this step.
+If you are installing by using Helm charts, do not perform this step. Complete the installation by using Helm; See "Installing
+{product-title-short} on secured clusters by using Helm charts" in the additonal resources section.
```
Comment (kcarmichael08) [inline modules/create-resource-init-bundle.adoc:54]:
```suggestion
{product-title-short} on secured clusters by using Helm charts" in the additional resources section.
```

**Review by kcarmichael08 — COMMENTED** [review]
one suggestion; otherwise lgtm!

File: modules/create-resource-init-bundle.adoc (hunk 1 above)
Comment (subhtk) [reply modules/create-resource-init-bundle.adoc:54]: Ack.

**Comment by subhtk** [comment]
@kcarmichael08 Done with the change.

**Comment by kcarmichael08** [comment]
I think the only things left to do are to edit the body of the PR to include the preview link and I would also put a note next to the QE box saying no QE is needed since it is just adding a link.

**Comment by subhtk** [comment]
@kcarmichael08 Please go ahead with the merging.

**Pull request merged**